RadixWallet/Clients/SensitiveInfoClient/SensitiveInfoClient+Live.swift

```diff
@@ -0,0 +1,19 @@
+extension SensitiveInfoClient: DependencyKey {
+	static var liveValue: SensitiveInfoClient {
+		.init(
+			read: { key in
+				guard let filePath = Bundle.main.path(forResource: "SensitiveInfo", ofType: "plist") else {
+					fatalError("Couldn't find file 'SensitiveInfo.plist'")
+				}
+				let plist = NSDictionary(contentsOfFile: filePath)
+				guard let value = plist?.object(forKey: key.rawValue) as? String else {
+					fatalError("Couldn't find value for key '\(key.rawValue)' in 'SensitiveInfo.plist'.")
```

Comment:
`fatalError` is intentional?

Reply:
Will change to `assertionFailure` instead